src/index.html

```diff
@@ -86,7 +86,7 @@
     <!-- <script src="//localhost:35729/livereload.js"></script> -->
 
     <!-- all the scripts -->
-    <script id="main-script" defer module type="text/javascript" src="./js/enketo-bundle.js" charset="utf-8"></script>
+    <script id="main-script" defer module type="text/javascript" src="./js/app.js" charset="utf-8"></script>
```

Comment:
@MartijnR, when you get a moment, would be great to get a sanity check on this change and whether you know of contexts in which it could cause problems.

This is a companion to https://github.com/enketo/enketo-express/pull/289 and I'm merging now after letting it sit for a few days to avoid conflicts and start getting the benefits to development.